main.go

```
@@ -0,0 +1,21 @@
+
+	"github.com/pajdekpl/gator/internal/config"
+)
+
+func main() {
+	fmt.Println("Hello world")
+	config, err := config.Read()
+	if err != nil {
+		log.Fatalf("error during reading config %v", err)
+	}
+	fmt.Printf("Read config: %+v\n", config)
+	config.SetUser("pajdek")
```

golangci-lint (1.62.2)

Error return value of `config.SetUser` is not checked

(errcheck)